### PR TITLE
Allow keepAliveIntervalInSec to be configured when calling aws_iot_shadow_connect

### PR DESCRIPTION
**Pull request by pctj101**
I have issues with the keep alive interval being "too long" which causes my SSL tunnel to close down.

It would be nicer to have keepAliveIntervalInSec be configurable even when using shadows.

### CODE REVIEW

**Comment by vareddy** [comment]
Hi @pctj101,
Thank you for your pull request! We will merge it into mainline after the next release. 
Please let us know if you have any more questions.
Thanks!
Varun

File: src/aws_iot_shadow.c [hunk 1]
```diff
@@ -33,10 +33,10 @@ extern "C" {
 #include "aws_iot_shadow_records.h"
 
 const ShadowInitParameters_t ShadowInitParametersDefault = {(char *) AWS_IOT_MQTT_HOST, AWS_IOT_MQTT_PORT, NULL, NULL,
-															NULL, false, NULL};
+															NULL, false};
```
Comment (gordonwang0) [inline src/aws_iot_shadow.c]: I think there should be a NULL after the false here to initialize `ShadowInitParametersDefault.disconnectHandler`.  Can you take a look at this?

Comment (escherstair) [reply src/aws_iot_shadow.c]: I cannot speak for @pctj101 but I think the `NULL` is necessary.

**Comment by gordonwang0** [comment]
Hello @pctj101,

I am closing this PR due to age, and so we may focus our development efforts on version 4.  Please let me know if you have any questions.

**Pull request closed without merging**